docs/_data/portland-2021-config.yaml

```diff
@@ -180,6 +180,14 @@ sponsors:
       link: https://www.knowledgeowl.com
       brand: KnowledgeOwl
       width: 250px
+    - name: juniper
+      link: https://www.juniper.net/
+      brand: Juniper
+      text: "<p><b>Power connections. Empower change.</b>\n Juniper exists to solve the world’s most difficult problems in networking. We do this by remaining true to our values of being bold, building trust and delivering excellence. Visit us at the virtual job fair!</p>"
```

Comment:
Does this text display actually work? Don't see it on https://writethedocs-www--1456.org.readthedocs.build/conf/portland/2021/sponsors/

Reply:
It should, I see it there.